File: sa31/2024-11-06/project/JavaLesson1/src/local/hal/sa31/java/lesson1/ex1/BasicSyntax.java
package local.hal.sa31.java.lesson1.ex1;

/**
 * SA31 Java文法 Lesson1 Ex1
 *
 * 実行クラス
 *
 * @author Shinzo SAITO
 */
public class BasicSyntax {
    public static void main(String[] args) {
        int score = (int) Math.round(Math.random() * 100);
        System.out.println("あなたの点数: " + score);
        String msg;
        if (score >= 80) {
            msg = "優です。";
        }
        else if (score >= 70) {
            msg = "良です。";
        }
        else if (score >= 60) {
            msg = "可です。";
        }
        else {
            msg = "不可です。";
        }
        System.out.println(msg);
//        msg = 35;
    }
}

File: sa31/2024-11-06/project/JavaLesson1/src/local/hal/sa31/java/lesson1/ex1/basicSyntax.php
<?php
/**
 * SA31 Java文法 Lesson1 Ex1
 *
 * PHP実行ファイル
 *
 * @author Shinzo SAITO
 */
$score = rand(0, 100);
print("あなたの点数: ".$score."<br>");
$msg = "";
if (score >= 80) {
    $msg = "優です。";
}
else if (score >= 80) {
    $msg = "良です。";
}
else if (score >= 80) {
    $msg = "可です。";
}
else {
    $msg = "不可です。";
}
print($msg);
// $msg = 35;